html5lib/html5parser.py

```diff
@@ -22,18 +22,18 @@
 
 
 def parse(doc, treebuilder="etree", encoding=None,
-          namespaceHTMLElements=True):
+          namespaceHTMLElements=True, script=True):
```

Comment:
Also, I'd rather we default to False, given we don't actually implement scripting.